ui/page/main_page.go

```diff
@@ -46,11 +46,19 @@ type NavHandler struct {
 
 type MainPage struct {
 	*load.Load
-	appBarNavItems          []NavHandler
-	drawerNavItems          []NavHandler
-	isNavDrawerMinimized    bool
+	appBarNavItems []NavHandler
+	drawerNavItems []NavHandler
+
+	axis      layout.Axis
+	textSize  unit.Value
+	leftInset unit.Value
+	width     unit.Value
+	alignment layout.Alignment
+	direction layout.Direction
```

Comment:
Create a NavDrawer struct in the components package to house the nav handlers and these properties.